File: app/src/main/java/com/qingshuige/tangyuan/models/UserProfile.java
package com.qingshuige.tangyuan.models;


/**
 * 用户资料
 */
public class UserProfile {
    /**
     * 用户唯一ID
     */
    private int userId;

    /**
     * 用户昵称
     */
    private String nickname;

    /**
     * 用户个性签名
     */
    private String bio;

    public int getUserId() {
        return userId;
    }

    public String getNickname() {
        return nickname;
    }

    public String getBio() {
        return bio;
    }
}

File: app/src/main/java/com/qingshuige/tangyuan/viewmodels/UserProfile.kt
package com.qingshuige.tangyuan.viewmodels


/**
 * 用户资料
 */
data class UserProfile(
    /**
     * 用户唯一 ID
     */
    val userId: Int = 0,

    /**
     * 用户昵称
     */
    val nickname: String? = null,

    /**
     * 用户个性签名
     */
    val bio: String? = null
)
